app/src/main/kotlin/com/mapbox/vision/teaser/MainActivity.kt

```diff
@@ -337,6 +325,11 @@ class MainActivity : AppCompatActivity(), ReplayModeFragment.OnSelectModeItemLis
         showSegmentationDetectionFragment(stateLoss)
     }
 
+    private fun showLaneDetectionFragment(stateLoss: Boolean = false) {
+        vision_view.visualizationMode = VisualizationMode.LaneDetection
+        showSegmentationDetectionFragment(stateLoss)
```

Comment:
Let's rename this fragment, since it's not SegmentationDetection anymore

Reply:
Thought about it.
However LaneDetection is also Detection, isn't it?
@yuryybk @yunikkk - I am open to new name proposals.

Reply:
Detection has a strict association with object detection like a bounding box.
Probably **DetectionSegmentationLaneFragment** (DSL) ? :)

Reply:
![image](https://user-images.githubusercontent.com/15800566/85005788-edee8f80-b161-11ea-9f53-6ec45c7fedb7.png)

Reply:
I'd probably go with

> However LaneDetection is also Detection, isn't it?

Well, I'd say it's a coincidence and similar wording =) 


> Probably **DetectionSegmentationLaneFragment** (DSL) ? :)

IMO it'll confuse us, since this is a term from ML world, one can suggest that results are based _solely_ on the DSL model, which is wrong - lane detection is a result of algos + model results.


I'd probably return old good LaneFragment, I've added the comment about it previously

Reply:
You mean create new fragment which will be absolutely the same as `SegmentationFragment`?
Logic for drawing any `VisualizationMode` is the same, I see no sense in having 2 fragments.

Reply:
No, and seems my comment about LaneFragment had been removed. 
I think we shouldn't remove the lane indicator that was present previously at all. 
First it allows users to check the usage of LaneDetection apis. 
Second it'll show multiple lanes if we finally will return multiple detected lanes

Reply:
Mmm, I see what you mean.
I like it - will do!